lib/wasix/src/fs/fd_list.rs

```diff
@@ -83,21 +83,21 @@ impl FdList {
         }
     }
 
-    pub fn insert_first_free_after(&mut self, fd: Fd, after: WasiFd) -> WasiFd {
+    pub fn insert_first_free_after(&mut self, fd: Fd, after_or_equal: WasiFd) -> WasiFd {
         match self.first_free {
             // We're shorter than `after`, need to extend the list regardless of whether we have holes
-            _ if self.fds.len() < after as usize => {
-                if !self.insert(true, after, fd) {
-                    panic!("Internal error in FdList - expected {after} to be unonccupied since the list wasn't long enough");
+            _ if self.fds.len() < after_or_equal as usize => {
+                if !self.insert(true, after_or_equal, fd) {
+                    panic!("Internal error in FdList - expected {after_or_equal} to be unonccupied since the list wasn't long enough");
```

Comment:
Just a typo here.

```suggestion
                    panic!("Internal error in FdList - expected {after_or_equal} to be unoccupied since the list wasn't long enough");
```